components/form-builder/app/edit/elements/lexical-editor/Toolbar.tsx

```diff
@@ -76,8 +76,6 @@ export const Toolbar = ({
 
   const [isEditable] = useState(() => editor.isEditable());
 
-  const { status: formBuilderDebugEnabled } = useFlag("formBuilderDebug");
-
   const toggleTreeView = () => {
```

Comment:
Noticed a few linting errors around unused vars since the TreeView functionality was removed.